static/js/manage/backlog-overview.js

```diff
@@ -1,4 +1,59 @@
 MEDUSA.manage.backlogOverview = function() {
+
+    $(document).ready(function() {
+        checkManualSearches();
+    });
+
+    function checkManualSearches() {
+        var pollInterval = 5000;
+        var searchStatusUrl = 'home/getManualSearchStatus';
+        var showId = $('#showID').val();
+        var url = showId === undefined ? searchStatusUrl : searchStatusUrl + '?show=' + showId;
+        $.ajax({
+            url: url,
+            error: function() {
+                pollInterval = 30000;
+            },
+            type: 'GET',
+            dataType: 'JSON',
+            complete: function() {
+                setTimeout(checkManualSearches, pollInterval);
+            },
+            timeout: 15000 // timeout every 15 secs
+        }).done(function(data) {
+            if (data.episodes) {
+                pollInterval = 5000;
+            } else {
+                pollInterval = 15000;
+            }
+            updateForcedSearch(data);
+        });
+    }
+
+    function updateForcedSearch(data) {
+        $.each(data.episodes, function(name, ep) {
+            var el = $('a[id=' + ep.show + 'x' + ep.season + 'x' + ep.episode + ']');
+            var img = el.children('img[data-ep-search]');
+            var parent = el.parent();
+            if (el) {
+                if (ep.searchstatus.toLowerCase() === 'searching' || ep.searchstatus.toLowerCase() === 'queued') {
```

Comment:
Why not just save `ep.searchStatus.toLowerCase()` to a var?
Also you can use data attributes so you don't have to parse the text. This will future proof it for when we add in translations.